dashboards/resources/node.libsonnet

```diff
@@ -44,8 +44,29 @@ local template = grafana.template;
         g.row('CPU Usage')
         .addPanel(
           g.panel('CPU Usage') +
-          g.queryPanel('sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{%(clusterLabel)s="$cluster", node=~"$node"}) by (pod)' % $._config, '{{pod}}') +
-          g.stack,
+          g.queryPanel([
+            'sum(kube_node_status_capacity{%(clusterLabel)s="$cluster", node=~"$node", resource="cpu"})' % $._config,
+            'sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{%(clusterLabel)s="$cluster", node=~"$node"}) by (pod)' % $._config,
```

Comment:
The order matters here! During making screenshots available in PR description this order was inverted (`kube_node_satatus_capacity`) was second. This resulted in the legend that had `max capacity` at the end of the list. However inverting this and making series with `kube_node_satatus_capacity` first improves UX by making `max capacity` appear first in the legend list.